gym/wrappers/pixel_observation.py

```diff
@@ -80,6 +80,7 @@ def __init__(
 
         # Extend observation space with pixels.
 
+        self.env.reset()
```

Comment:
I guess this is necessary to get the shape of the image? Is there any reasonable way to skip the reset? (my intuition is that there isn't, it's fine if that's just the answer)

Reply:
The assumption is that the environment is generated using `gym.make` therefore, we will need to reset before rendering. Also this was failing the tests without it.